src/Core/WhenExpr/ContextKeys.re

```diff
@@ -0,0 +1,59 @@
+module Lookup = Kernel.KeyedStringMap;
+
+module Value = {
+  [@deriving show({with_path: false})]
+  type t =
+    | String(string)
+    | True
+    | False;
+
+  // Emulate JavaScript semantics
+  let asBool =
+    fun
+    | True => true
+    | False => false
+    | String("") => false
+    | String(_) => true;
+
+  // Emulate JavaScript semantics
+  let asString =
```

Comment:
Cool, we support string context now!

Reply:
Is there any context key we're using this now? Or is this in preparation for the additional SCM functionality?

Reply:
This was added when I revamped `WhenExpr`. It's just been moved here. It's currently just used in unit tests, but will be needed for the SCM functionality shortly.